udata/settings.py

```diff
@@ -37,6 +37,8 @@ class Defaults(object):
     CACHE_KEY_PREFIX = 'udata-cache'
     CACHE_TYPE = 'redis'
 
+    SERVER_NAME = 'localhost:7000'
```

Comment:
I wonder if it might not be better in `Debug` settings as it's requirement to set this value in production; (Open to debate, just wondering)